### ISSUE TITLE
v3: Cannot query field "description" on type "PrismicEmbedType"

### ISSUE THREAD

**Issue by kieranstartup** [angeloashmore/gatsby-source-prismic#225]
Hey,

I'm working on a project where I need to display the `description` field for a Vimeo embed but the field isn't available when I query for it. 

I've taken a look at what fields are returned from an oembed field using the Prismic GraphiQL viewer and description is returned, so I'm guessing it's just not accessible via the plugin?

Here's all the fields outputted from Prismic's GraphiQL viewer if it is of use:

````
  "video": {
    "is_plus": "0",
    "account_type": "pro",
    "width": 426,
    "height": 240,
    "duration": 83,
    "description": "Vimeo Video Description",
    "thumbnail_url_with_play_button": "https://i.vimeocdn.com/filter/overlay?src0=https%3A%2F%2Fi.vimeocdn.com%2Fvideo%2F856130533_295x166.jpg&src1=http%3A%2F%2Ff.vimeocdn.com%2Fp%2Fimages%2Fcrawler_play.png",
    "upload_date": "2020-02-14 08:10:38",
    "video_id": 391482436,
    "uri": "/videos/391482436:c6df2d1cf4",
    "embed_url": "https://vimeo.com/391482436/c6df2d1cf4",
    "type": "video",
    "version": "1.0",
    "title": "Vimeo Video Title",
    "author_name": "Vimeo Author Name",
    "author_url": "https://vimeo.com/user108479578",
    "provider_name": "Vimeo",
    "provider_url": "https://vimeo.com/",
    "cache_age": null,
    "thumbnail_url": "https://i.vimeocdn.com/video/856130533_295x166.jpg",
    "thumbnail_width": 295,
    "thumbnail_height": 166,
    "html": "<iframe src=\"https://player.vimeo.com/video/391482436?app_id=122963\" width=\"426\" height=\"240\" frameborder=\"0\" allow=\"autoplay; fullscreen\" allowfullscreen title=\"198_Apollo_Project\"></iframe>"
  }
```` 

Thanks!

**Comment by angeloashmore** [comment]
Thanks, published a fix for the description field on `v3.0.2`.

It's becoming clear an embed field's data is dynamic based on the type of embed, so we should probably allow Gatsby to infer the types. I'll open a new issue to track it.

**Comment by kieranstartup** [comment]
Amazing, thanks for pushing a fix for that so quickly. Much appreciated!
